pkg/core/blockchain.go

```diff
@@ -83,11 +83,11 @@ func NewBlockchain(ctx context.Context, s storage.Store, cfg config.ProtocolConf
 	return bc, nil
 }
 
-// NewBlockchainLevelDB initializes new blockchain DB store based on configuration
-func NewBlockchainLevelDB(ctx context.Context, cfg config.Config) (*Blockchain, error) {
+// NewLevelDBBlockChain initializes new blockchain DB store based on configuration
+func NewLevelDBBlockChain(ctx context.Context, cfg config.Config) (*Blockchain, error) {
```

Comment:
Maybe it's better to move these functions into the `storage`? You just need to make them having a uniform interface like `f(context.Context, config.Config) (*Store, error)` (third parameter for `storage.NewLevelDBStore` shouldn't exist anyway) and then all of these `New*BlockChain` functions are gonna be almost the same, error handling and `NewBlockhain()` invocation could then be moved into `initBlockchainDB()`.